spec/models/object_event_hook_config_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/master/LICENSE
+require 'rails_helper'
+
+RSpec.describe ObjectEventHookConfig, type: :model do
+  let(:nonprofit) { create(:nm_justice) }
+  let(:open_fn_config) { create(:open_fn_config, nonprofit_id: nonprofit.id) }
```

Comment:
I don't know if I like this, but I wasn't able to do `nonprofit.object_event_hook_configs.create(:open_fn_config)` using the factory. Do you have any ideas on how to make it look better?

Reply:
Why weren't you able to it using the factory?

Reply:
Oh, I know what you mean now.

Reply:
This is fine. There's a lot of work that should be done to simplify setting things up from factories but those are future things that I should document :)